.template/ui/config/chainConfig.json

```diff
@@ -0,0 +1,51 @@
+{
+    "chain_name": "local_testnet",
+    "block_production": {
+      "ProofOfAuthority": {
+        "trigger": "instant"
+      }
+    },
+    "parent_network": {
+      "type": "LocalTest"
+    },
+    "block_gas_limit": 1000000000,
+    "initial_state": {
+      "coins": [
+        {
+          "owner": "0x94ffcc53b892684acefaebc8a3d4a595e528a8cf664eeb3ef36f1020b0809d0d",
+          "amount": "0xFFFFFFFFFFFFFFFF",
+          "asset_id": "0x0000000000000000000000000000000000000000000000000000000000000000"
+        },
+        {
+          "owner": "0x80d5e88c2b23ec2be6b2e76f3499a1a2755bb2773363785111a719513fb57b8e",
+          "amount": "0x00000000FFFFFFFF",
+          "asset_id": "0x0000000000000000000000000000000000000000000000000000000000000000"
+        },
+        {
+          "owner": "0xf13c949256d0e119fecaec414ea452f21f9dc1870fb6262ff53b37c32cab4749",
+          "amount": "0x00000000FFFFFFFF",
+          "asset_id": "0x0000000000000000000000000000000000000000000000000000000000000000"
+        }
+      ]
+    },
+    "transaction_parameters": {
+      "contract_max_size": 16777216,
+      "max_inputs": 255,
+      "max_outputs": 255,
+      "max_witnesses": 255,
+      "max_gas_per_tx": 100000000,
+      "max_script_length": 1048576,
+      "max_script_data_length": 1048576,
+      "max_static_contracts": 255,
+      "max_storage_slots": 255,
+      "max_predicate_length": 1048576,
+      "max_predicate_data_length": 1048576,
+      "gas_price_factor": 1000000000,
+      "gas_per_byte": 4,
+      "max_message_data_length": 1048576
+    },
+    "block_producer": {
+      "utxo_validation": true,
+      "coinbase_recipient": "0x0000000000000000000000000000000000000000000000000000000000000000"
+    }
+  }
```

Comment:
Does this work? It's missing:
```
  "consensus": {
    "PoA": {
      "signing_key": "0x22ec92c3105c942a6640bdc4e4907286ec4728e8cfc0d8ac59aad4d8e1ccaefb"
    }
  }
```

Reply:
No idea, in the description I've mentioned that I don't know what to put so I've taken this from somewhere, likely the TS quickstart.

Reply:
1. When I tried to run a node with a chain config without that field, I got "consensus field is missing" kind of error. That's why I asked if it's working without it. Did you try?
2. Did you see my previous comment about a default chain config? TS docs are not updated everytime there's an update to the core.

Reply:
1. Tried it now with your suggestion and it runs.
2. Yes

Reply:
1. I simply wonder if it is _not running_ without the "consensus" field. 
2. I would suggest double-checking what `ChainConfig::default()` returns before merging this.

Reply:
1. You have confirmed that it doesn't run without it. I have removed the field and it gave me the same error so it is needed.
2. "I didn't feel like digging into core to figure out what to run / call in order to debug to see what the default produced". If you'd like to note the steps of what you did and where to get that output then I can try it otherwise this new default runs.

Reply:
> I have removed the field and it gave me the same error

This is what I was asking for.

> If you'd like to note the steps of what you did and where to get that output

I wanted to double-check because the node running does not convince me that there won't be consensus issues. I noted the steps in my comment above, that's why I keep pointing to that. I'll quote my comment here (and I did it again already so you don't have to):

> Since there is no documentation, I call `ChainConfig::default()` and debug (the dependency for that is `fuel-core-chain-config = "0.17"` as of now).

So I go to a Rust file like a test for a contract, I add `fuel-core-chain-config = "0.17"` to dependencies, go to the file that I want to see running, add `use fuel_core_chain_config::ChainConfig` to the top, go to a function and call `dbg!(ChainConfig::default())`

What I get in the terminal after I `cargo test -- --nocapture` is this:

```
ChainConfig::default() = ChainConfig {
    chain_name: "local",
    block_gas_limit: 1000000000,
    initial_state: None,
    transaction_parameters: ConsensusParameters {
        contract_max_size: 16777216,
        max_inputs: 255,
        max_outputs: 255,
        max_witnesses: 255,
        max_gas_per_tx: 100000000,
        max_script_length: 1048576,
        max_script_data_length: 1048576,
        max_storage_slots: 255,
        max_predicate_length: 1048576,
        max_predicate_data_length: 1048576,
        gas_price_factor: 1000000000,
        gas_per_byte: 4,
        max_message_data_length: 1048576,
    },
    gas_costs: GasCosts(
        GasCostsValues {
            add: 1,
            addi: 1,
            aloc: 1,
            and: 1,
            andi: 1,
            bal: 21,
            bhei: 1,
            bhsh: 1,
            burn: 35,
            cb: 2,
            cfei: 1,
            cfsi: 1,
            croo: 28,
            div: 1,
            divi: 1,
            ecr: 1703,
            eq: 1,
            exp: 1,
            expi: 1,
            flag: 1,
            gm: 1,
            gt: 1,
            gtf: 1,
            ji: 1,
            jmp: 1,
            jne: 1,
            jnei: 1,
            jnzi: 1,
            k256: 19,
            lb: 1,
            log: 40,
            lt: 1,
            lw: 1,
            mcpi: 3,
            mint: 35,
            mlog: 1,
            mod_op: 1,
            modi: 1,
            move_op: 1,
            movi: 1,
            mroo: 2,
            mul: 1,
            muli: 1,
            noop: 1,
            not: 1,
            or: 1,
            ori: 1,
            ret: 61,
            rvrt: 61,
            s256: 5,
            sb: 1,
            scwq: 11,
            sll: 1,
            slli: 1,
            srl: 1,
            srli: 1,
            srw: 23,
            sub: 1,
            subi: 1,
            sw: 1,
            sww: 79,
            swwq: 72,
            time: 1,
            tr: 120,
            tro: 99,
            xor: 1,
            xori: 1,
            call: DependentCost {
                base: 116,
                dep_per_unit: 14,
            },
            ccp: DependentCost {
                base: 24,
                dep_per_unit: 13,
            },
            csiz: DependentCost {
                base: 17,
                dep_per_unit: 15,
            },
            ldc: DependentCost {
                base: 23,
                dep_per_unit: 14,
            },
            logd: DependentCost {
                base: 46,
                dep_per_unit: 19,
            },
            mcl: DependentCost {
                base: 1,
                dep_per_unit: 2359,
            },
            mcli: DependentCost {
                base: 1,
                dep_per_unit: 2322,
            },
            mcp: DependentCost {
                base: 1,
                dep_per_unit: 1235,
            },
            meq: DependentCost {
                base: 1,
                dep_per_unit: 2343,
            },
            retd: DependentCost {
                base: 65,
                dep_per_unit: 19,
            },
            smo: DependentCost {
                base: 84,
                dep_per_unit: 13,
            },
            srwq: DependentCost {
                base: 54,
                dep_per_unit: 2,
            },
        },
    ),
    consensus: PoA {
        signing_key: 0x22ec92c3105c942a6640bdc4e4907286ec4728e8cfc0d8ac59aad4d8e1ccaefb,
    },
}
```

Reply:
> So I go to a Rust file like a test for a contract, I add fuel-core-chain-config = "0.17" to dependencies, go to the file that I want to see running, add use fuel_core_chain_config::ChainConfig to the top, go to a function and call dbg!(ChainConfig::default())
>
> What I get in the terminal after I cargo test -- --nocapture is this:

This makes more sense. I don't know the difference between the current config and the default. The current one seems to run and it includes funding afaik. I'm unsure if we should use the default here.